youtube_dl/extractor/indavideo.py

```diff
@@ -9,8 +11,14 @@
 )
 
 
-class IndavideoEmbedIE(InfoExtractor):
```

Comment:
I've merged the two way of extractions (download and embedding). I think this general name (without embed) describes better the function. So I didn't chane the name, I've only deleted the Embeded function, merging its functionality to the other one.